File: CS/WindowsApplication1/MyXtraTabbedMdiManager.cs
using System;
using System.Drawing;
using System.Windows.Forms;
using DevExpress.XtraTabbedMdi;
using DevExpress.Utils.Controls;
using DevExpress.Utils;

namespace WindowsApplication1
{
    public class MyXtraTabbedMdiManager : XtraTabbedMdiManager
    {

        public MyXtraTabbedMdiManager()
        {
            timer.Interval = 500;
            timer.Tick += new EventHandler(timer_Tick);
        }

        void timer_Tick(object sender, EventArgs e)
        {
            Point mousePosition = ViewInfo.TabControl.ScreenPointToControl(Control.MousePosition);
            if (InButtons(mousePosition))
                ViewInfo.HeaderInfo.Buttons.ProcessEvent(new ProcessEventEventArgs(EventType.MouseDown, new DXMouseEventArgs(MouseButtons.Left, 1, mousePosition.X, mousePosition.Y, 0)));
        }

        private bool InButtons(Point location)
        {
            DevExpress.XtraTab.ViewInfo.BaseTabHitInfo hi = CalcHitInfo(location);
            return hi.HitTest == DevExpress.XtraTab.ViewInfo.XtraTabHitTest.PageHeaderButtons;
        }

        protected override void OnMouseDown(DevExpress.Utils.DXMouseEventArgs e)
        {
            base.OnMouseDown(e);
            if (InButtons(e.Location))
                StartRepeat();
        }

        protected override void OnMouseUp(DevExpress.Utils.DXMouseEventArgs e)
        {
            base.OnMouseUp(e);
            EndRepeat();
        }
        Timer timer = new Timer();

        private void StartRepeat()
        {
            timer.Enabled = true;
        }
        private void EndRepeat()
        {
            timer.Enabled = false;
        }

    }
}

File: VB/WindowsApplication1/MyXtraTabbedMdiManager.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Drawing
Imports System.Windows.Forms
Imports DevExpress.XtraTabbedMdi
Imports DevExpress.Utils.Controls
Imports DevExpress.Utils

Namespace WindowsApplication1
	Public Class MyXtraTabbedMdiManager
		Inherits XtraTabbedMdiManager

		Public Sub New()
			timer.Interval = 500
			AddHandler timer.Tick, AddressOf timer_Tick
		End Sub

		Private Sub timer_Tick(ByVal sender As Object, ByVal e As EventArgs)
			Dim mousePosition As Point = ViewInfo.TabControl.ScreenPointToControl(Control.MousePosition)
			If InButtons(mousePosition) Then
				ViewInfo.HeaderInfo.Buttons.ProcessEvent(New ProcessEventEventArgs(EventType.MouseDown, New DXMouseEventArgs(MouseButtons.Left, 1, mousePosition.X, mousePosition.Y, 0)))
			End If
		End Sub

		Private Function InButtons(ByVal location As Point) As Boolean
			Dim hi As DevExpress.XtraTab.ViewInfo.BaseTabHitInfo = CalcHitInfo(location)
			Return hi.HitTest = DevExpress.XtraTab.ViewInfo.XtraTabHitTest.PageHeaderButtons
		End Function

		Protected Overrides Sub OnMouseDown(ByVal e As DevExpress.Utils.DXMouseEventArgs)
			MyBase.OnMouseDown(e)
			If InButtons(e.Location) Then
				StartRepeat()
			End If
		End Sub

		Protected Overrides Sub OnMouseUp(ByVal e As DevExpress.Utils.DXMouseEventArgs)
			MyBase.OnMouseUp(e)
			EndRepeat()
		End Sub
		Private timer As New Timer()

		Private Sub StartRepeat()
			timer.Enabled = True
		End Sub
		Private Sub EndRepeat()
			timer.Enabled = False
		End Sub

	End Class
End Namespace
